web/src/components/ak-wizard-main/ak-wizard-frame.ts

```diff
@@ -162,47 +177,32 @@ export class AkWizardFrame extends CustomEmitterElement(ModalButton) {
 
     renderFooter() {
         return html`
-            <footer class="pf-c-wizard__footer">${map(this.buttons, this.renderButtons)}</footer>
+            <footer class="pf-c-wizard__footer">${map(this.buttons, this.renderButton)}</footer>
         `;
     }
 
-    renderButtons([label, command]: WizardButton) {
```

Comment:
Didn't need both functions; just a map and a way to tell if the button is disabled.